layouts/index.html

```diff
@@ -4,7 +4,7 @@
     var animation = false;
     var availableCommands = ["all", "whois", 'social', 'work', 'education',
                     'skills', 'softskills', 'languages',
-                    'projects', 
+                    'projects' 'certificates', 
```

Comment:
* Missing a comma (,)
* Maybe "Certifications" Is a better word for this field?? I am not native English speaker, therefore can't tell for sure and I am not picky therefore will leave this one up to you any of them its fine by me. 
* Some users might not have certificates, therefore lets make this one optional, and use the same method and code style as the options below. 

```
 ... 
 projects,
 {{ if .Site.Params.certificates }} 'certificates', {{ end }}
 {{ if .Site.Params.startxLocation }} 'startx', {{ end }}
 ...
 ```

Reply:
No worries!

Certificates should certainly be optional because as you said some users might not have them and I think either term is fine but I think certifications definitely sounds more professional and formal